src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLStopImpl.java

```diff
@@ -83,7 +84,19 @@
 
   @Override
   public DataFetcher<String> name() {
-    return environment -> getValue(environment, StopLocation::getName, Station::getName);
+    return environment -> {
+      String lang = new LegacyGraphQLTypes.LegacyGraphQLStopNameArgs(
+              environment.getArguments()).getLegacyGraphQLLang();
+      return getValue(
+              environment,
+              stop -> {
+                return stop.getName().toString(lang == null ? environment.getLocale() : new Locale(lang));
```

Comment:
I think there is quite a bit of duplication here and in the URL. Can you make this a little DRY?

Reply:
This uses `LegacyGraphQLUtils.getLocale` now